collectors/python.d.plugin/tomcat/tomcat.chart.py

```diff
@@ -105,8 +105,11 @@ def _get_data(self):
         data = None
         raw_data = self._get_raw_data()
         if raw_data:
-            # Fix attributes ending with single quote content
-            raw_data = re.sub(r"='([^']+)'([^']+)''", r"='\g<1>\g<2>'", raw_data)
+            # Fix XML attributes ending with single quote content, ie. <memorypool name='CodeHeap 'non-nmethods'' ... />
+            # Cf. Issue https://github.com/netdata/netdata/issues/6343
+            pattern = re.compile(r"='([^']+)'([^']+)''")
```

Comment:
move compile to the global area, we dont want compile it on every iteration (on every get_data call)

Reply:
Oh yeah I see, my bad.